.eslintrc.json

```diff
@@ -101,7 +101,8 @@
       2,
       {
         "code": 100,
-        "ignorePattern": "(https?:|JSON\\.parse|[Uu]rl =)"
+        "ignorePattern": "(https?:|JSON\\.parse|[Uu]rl =)",
+        "ignoreStrings": true
```

Comment:
If a line is too long because it contains a long string, don't raise an eslint error.